cosmos/ssc.json

```diff
@@ -0,0 +1,50 @@
+{
+  "chainId": "ssc-1",
+  "chainName": "Saga Mainnet",
+  "rpc": "https://ssc-rpc.sagarpc.io",
+  "rest": "https://ssc-lcd.sagarpc.io",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ssc-mainnet/chain.png",
+  "bip44": {
+    "coinType": 118
+  },
+  "bech32Config": {
+    "bech32PrefixAccAddr": "saga",
+    "bech32PrefixAccPub": "sagapub",
+    "bech32PrefixValAddr": "sagavaloper",
+    "bech32PrefixValPub": "sagavaloperpub",
+    "bech32PrefixConsAddr": "sagavalcons",
+    "bech32PrefixConsPub": "sagavalconspub"
+  },
+  "currencies": [
+    {
+      "coinDenom": "SAGA",
+      "coinMinimalDenom": "usaga",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ssc-mainnet/chain.png"
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "SAGA",
+      "coinMinimalDenom": "usaga",
+      "coinDecimals": 6,
+      "gasPriceStep": {
+        "low": 0.0025,
+        "average": 0.025,
+        "high": 0.04
+      },
```

Comment:
@ashishchandr70 Could you check gas price step?
I have heard that it is recommended to set the low price of the validator to 0.01.
This may prevent transactions sent with low price from being executed.

```
  low: 0.01,
  average: 0.025,
  high: 0.04,
  ```